artemis-protocols/artemis-amqp-protocol/src/main/java/org/apache/activemq/artemis/protocol/amqp/proton/AMQPTunneledCoreLargeMessageReader.java

```diff
@@ -204,7 +206,7 @@ public Message readBytes(Delivery delivery) throws Exception {
             final Message result = coreLargeMessage.toMessage();
 
             // We don't want a close to delete the file now, so we release these resources.
-            coreLargeMessage.releaseResources(true, true);
+            coreLargeMessage.releaseResources(largeMessageSync, true);
```

Comment:
You don't need to track this value, just call the accessor in the Session context